sos/policies/suse.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, sysroot=None, init=None, probe_runtime=True,
         self.package_manager = PackageManager(
             'rpm -qa --queryformat "%{NAME}|%{VERSION}\\n"',
             remote_exec=remote_exec)
-        self.valid_subclasses = [SuSEPlugin, RedHatPlugin]
+        self.valid_subclasses += [SuSEPlugin]
```

Comment:
AFAIK, the SuSE policy was loading the `RedHatPlugin` tagging class as well on purpose, as there is a large overlap of compatibility there, and there are only 5 SuSE-tagged plugins to date.

Changing this would result in a loss of about 80 plugins on SuSE. I'd say for now let's keep the RH tag in the policy.